src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv/Internal/ListenerContext.cs

```diff
@@ -78,8 +78,7 @@ private UvStreamHandle AcceptHandle()
             switch (EndPointInformation.HandleType)
             {
                 case FileHandleType.Auto:
-                    //  Handle type detection must be done by listener, refreshing endpoint information
-                    throw new InvalidOperationException();
+                    throw new InvalidOperationException("Cannot accept on a non-specific file handle, listen should be performed first.");
```

Comment:
I considered making this localizable, but at least my copy of VS2017 15.3p3 doesn't handle .resx for Core as it does for Framework, hence the question -- should this be localizable?